subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt

```diff
@@ -210,7 +209,7 @@ abstract class InHouseInstrumentationTestRunner :
 
         testRunEnvironment.executeIfRealRun {
             Espresso.setFailureHandler(
-                ReportFriendlyFailureHandler(DefaultFailureHandler(targetContext))
```

Comment:
DefaultFailure handler does not do any improvements to stack traces.
But erase stack traces
`error.setStackTrace(Thread.currentThread().getStackTrace());`
Look at `androidx.test.espresso.base.DefaultFailureHandler#getUserFriendlyError` for details

Reply:
It'd good to file an issue about `DefaultFailureHandler`